test_to_do_app/test_integration.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+from dotenv import load_dotenv,find_dotenv
+from todo_app import app
+import requests
+from todo_app.data.trello_urls import TrelloUrls
+import os
+
+@pytest.fixture
+def client(monkeypatch):
+    file_path = find_dotenv('.env.test')
+    load_dotenv(file_path, override=True)
+    monkeypatch.setattr(requests,'get',get_lists_stub)
+    test_app = app.create_app()
+
+    with test_app.test_client() as client:
+        yield client
+
+def test_index_page(client):  
+    response = client.get('/')
+    assert response.status_code == 200
+    assert 'Card 1' in response.data.decode()
+
+class StubResponse():
+    def __init__(self, fake_response_data):
+        self.fake_response_data = fake_response_data
+
+    def json(self):
+        return self.fake_response_data
+
+def get_lists_stub(url, params):
+    NOT_STARTED_LIST_ID = os.environ.get('NOT_STARTED_LIST_ID')
+    STARTED_LIST_ID = os.environ.get('STARTED_LIST_ID')
+    DONE_LIST_ID = os.environ.get('DONE_LIST_ID')
+    trello_urls = TrelloUrls()
+    payload = trello_urls.auth.copy()
+    payload.update({'fields':['id','name','dateLastActivity']})
```

Comment:
In fact - is any of this bit needed? You set up the payload, but then you just ignore it and return fixed response data anyway; which is fine!
If we wanted to make this more specific then we might use something like this to check that the url/params being passed to the `requests.get` method (which is being replaced by this stub) are what we'd expect, but currently we aren't doing that check so there's no need for these that I can see